metric/system/process/process_test.go

```diff
@@ -118,6 +119,22 @@ func TestProcessList(t *testing.T) {
 	}
 }
 
+func TestSelfPersist(t *testing.T) {
+	stat, err := initTestResolver()
+	require.NoError(t, err, "Init()")
+	first, err := stat.GetSelf()
+	require.NoError(t, err, "First GetSelf()")
+
+	// The first process fetch shouldn't have percentages, since we don't have >1 procs to compare
```

Comment:
The code uses the delta between two measurements to calculate percentages. Therefore, the very first report we get will never have any percentages, as it will always save off the "last" value for calculation.